src/profile/ChangePassword.js

```diff
@@ -29,39 +27,42 @@ const ChangePassword = (props) => {
   };
 
   const handleChange = (event) => {
-    setstate({
+    setState({
       ...state,
-      [event.target.id]: event.target.value
+      [event.target.id]: event.target.value,
     });
   };
 
   const handleChangeClick = async (event) => {
     event.preventDefault();
-    setstate({
+    setState({
       ...state,
-      isChanging: true
+      isChanging: true,
     });
 
     try {
       const currentUser = await Auth.currentAuthenticatedUser();
-      await Auth.changePassword(currentUser,state.oldPassword,state.password);
+      await Auth.changePassword(
+        currentUser, 
+        state.oldPassword, 
+        state.password
+      );
       props.history.push("/settings");
-    } 
-    catch(e) {
+    } catch (e) {
       alert(e);
```

Comment:
Can you change this alert to an `errorToast`? You can import from the `src/libs/toast` library.

Reply:
I have added error toast function with the latest changes